lib/grape/api.rb

```diff
@@ -6,7 +6,7 @@ module Grape
   # should subclass this class in order to build an API.
   class API
     # Class methods that we want to call on the API rather than on the API object
-    NON_OVERRIDABLE = Class.new.methods.freeze
+    NON_OVERRIDABLE = Class.new.methods.freeze + %i[call call!]
```

Comment:
I think that this should be `(Class.new.methods + %i[call call!]).freeze`

Reply:
I was bored, https://github.com/ruby-grape/grape/pull/1837/files

Reply:
Nice!